common/model-views.cpp

```diff
@@ -2120,6 +2122,37 @@ namespace rs2
 
         ImGui::SetCursorPos({ stream_rect.w - 32 * num_of_buttons, 0 });
 
+        if (RS2_STREAM_DEPTH == profile.stream_type())
+        {
+            label = to_string() << textual_icons::bar_chart << "##Color map";
+            if (viewer.show_map_ruler)
```

Comment:
Why is this on the `viewer` object? Doesn't this mean the button controls all depth streams together?